keel-web/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -30,9 +37,15 @@ import org.springframework.stereotype.Component
  */
 @Component
 class ApplicationService(
-  private val repository: KeelRepository
+  private val repository: KeelRepository,
+  private val constraints: List<ConstraintEvaluator<*>>
```

Comment:
For naming consistency:
```suggestion
  private val constraintEvaluators: List<ConstraintEvaluator<*>>
```